manifests/cluster-network-addons/v0.42.0/operator.yaml

```diff
@@ -121,7 +121,7 @@ apiVersion: apps/v1
 kind: Deployment
```

Comment:
The directory structure and the version used in OLM is something we should not change. AFAIK OLM has issues with the `v` prefix.

We should keep it `v`-less everywhere except for git tags and image tags.

Reply:
Thanks for explanation, updated